src/main/java/com/ontrack/dto/TaskDTO.java

```diff
@@ -2,24 +2,24 @@
 
 public class TaskDTO {
 
-	private int taskId;
+	private int id;

```

Comment:
I renamed this to id since it is already inside the TaskDTO so it is already implied that this is a task id so rename it to id shortens the code and makes it more readable